CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - feat(workers): add 'nodeAssociatePublicIpAddress' to toggle public IPs
   [#81](https://github.com/pulumi/pulumi-eks/pull/81)
+- fix(getAmi): allow setting master version & explicitly filter Linux AMIs
```

Comment:
FWIW - allowing setting the master version is a meaningful feature addition in its own right, independent of fixing the AMI computation.  Might be worth two line items here.

Reply:
You got it 👍